jobs/dtd-metrics/src/index.ts

```diff
@@ -1,20 +1,21 @@
 import { ReadPreferenceMode } from 'mongodb'
 import { AwsS3BucketClient, ReadModelClient, withExecutionTime } from '@interop-be-reports/commons'
 import { env } from './configs/env.js'
-import { MetricsOutput, MetricsQueriesResult } from './models/metrics.model.js'
+import { MetricsOutput } from './models/metrics.model.js'
 import {
   getPublishedEServicesMetric,
   getEServicesByMacroCategoriesMetric,
   getMostSubscribedEServicesMetric,
   getTopProducersBySubscribersMetric,
-  getOnboardedTenantsCountMetric,
-  getTenantDistributionMetric,
-  getTenantSignupsTrendMetric,
-  getOnboardedTenantsCountByMacroCategoriesMetric,
   getTopProducersMetric,
+  // getOnboardedTenantsCountMetric,
+  // getTenantDistributionMetric,
+  // getTenantSignupsTrendMetric,
+  // getOnboardedTenantsCountByMacroCategoriesMetric,
```

Comment:
Will these be de-commented in the future?

Reply:
Yes, for now we decided to skip the computation of the second batch metrics

Reply:
We will de-comment all the related code when we will need them